web/app/components-v2/controls/Removable.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { t } from '../../i18n/i18n'
+import { common, CommonPropsWithChildren } from '../CommonProps'
+import { CHARCODE_REMOVE } from '../texts/Icon'
+import { IconButton } from './IconButton'
+
+export type RemovableProps = CommonPropsWithChildren<{
+  isRemovable?: boolean // default true
+  onClick: () => void
```

Comment:
Tällä sama homma mitä RaisedButton:ssa, eli `onClick`-handlerin tyypitys on hyvä tarkistaa siitä elementistä mihin `onClickiä` käytetään.

Reply:
Osassa vastaavia käyttötapauksia onClickillä yms. on oma sisäinen käsittelijänsä. Olisi varmaan käyttäjälle vähemmän yllättävä, että jos tehdään tämä muutos, niin tehdään sit kaikista käsittelijöistä samanlaisia – eli passataan alkuperäinen event aina eteenpäin. Ainakin helpottaisi tulevaisuudessa, jos pitää jossain tilanteessa esim. katkaista eventin kupliminen eteenpäin.